xml/System.Reflection/Assembly.xml

```diff
@@ -3076,7 +3076,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Is Assembly held in a collectible AssemblyLoadContext?</summary>
         <value>To be added.</value>
         <remarks>To be added.</remarks>
```

Comment:
it would be helpful to add a remark that explains what a collectible assembly load context is.

Reply:
I drafted the remarks